index.ts

```diff
@@ -1,35 +1,48 @@
-import express,{Express} from 'express'
-import {portNumber} from './src/constant/constants'
-import { userContext } from './src/constant/constants'
-import { userRouter } from './src/routers/router'
-import { mongo } from './src/provider/mongo/mongo'
-class App{
-    private app!: Express
-    private port!: number
-    private context!: string
-    constructor(){
-        this.startApp()
-    }
-    startApp(){
-        this.app=express()
-        this.loadGlobalMiddleWare()
-        mongo.initiateMongoConnection
-        this.loadRouter()
-        this.initServer()
-    }
-    loadGlobalMiddleWare() {
-        this.context=userContext
-        this.app.use(express.json())
-        this.port=portNumber
-    }
-    loadRouter() {
-        this.app.use(this.context,userRouter.userRouter())
-    }
-    initServer(){
-        this.app.listen(this.port,this.callback)
-    }
-    private callback = () => {
-        console.log(`Server listing on port: ${this.port}`);
-    };
+import express, { Express } from "express";
+import { portNumber } from "./src/constant/constants";
+import { userContext, swaggerContext, adminContext } from "./src/constant/constants";
+import { userRouter } from "./src/routers/router";
+import { adminRouterV1 } from "./src/routers/router";
+import { mongo } from "./src/provider/mongo/mongo";
+import { serve, setup } from "swagger-ui-express";
+import swaggerDocument from "./swagger/parse-swagger.json";
+import { utils } from "./src/utils/utils";
+class App {
```

Comment:
Use es-lint for linting the code. No linting is here, just after the import there is class. Add comments for each function and class.